src/main/java/org/apache/commons/validator/routines/ISBNValidator.java

```diff
@@ -220,7 +220,7 @@ public boolean isValidISBN13(final String code) {
      * Check the code is either a valid ISBN-10 or ISBN-13 code.
      * <p>
      * If valid, this method returns the ISBN code with
-     * formatting characters removed (i.e. space or hyphen).
+     * formatting characters removed (such as space or hyphen).
```

Comment:
are you sure other formatting characters than spaces or hyphens are indeed removed?

Reply:
Well, formatting characters are called SEP inside the regex pattern.
Based on the SEP constant it could be a hyphen or **any whitespace character**, not only space.
`    private static final String SEP = "(?:\\-|\\s)";`
As I know:
- Space ( )
- Tab (\t)
- Newline (\n)
- Carriage return (\r)
- Form feed (\f)

Reply:
Thanks for confirming, that means replacing `i.e.` with `such as` is a documentation bugfix in this case :+1: